### PR TITLE
doc: varargs and keywords

### PR DESCRIPTION
**Pull request by PaulSoderlind**
Small update of functions.md to clarify that `plot(x... ; style="solid")`works

### CODE REVIEW

File: doc/src/manual/functions.md [hunk 1]
```diff
@@ -525,6 +525,14 @@ Keyword argument default values are evaluated only when necessary (when a corres
 argument is not passed), and in left-to-right order. Therefore default expressions may refer to
 prior keyword arguments.
 
+Keyword arguments can also be used in varargs functions:
+
+```julia
+function plot(x... ; style="solid")
```
Comment (KristofferC) [inline doc/src/manual/functions.md]:
```suggestion
function plot(x...; style="solid")
```

File: doc/src/manual/functions.md [hunk 2]
```diff
@@ -347,7 +347,7 @@ not work.
 
 It is often convenient to be able to write functions taking an arbitrary number of arguments.
 Such functions are traditionally known as "varargs" functions, which is short for "variable number
-of arguments". You can define a varargs function by following the last argument with an ellipsis:
+of arguments". You can define a varargs function by following the last non-keyword argument with an ellipsis:
```
Comment (Keno) [inline doc/src/manual/functions.md]: Keyword arguments haven't been introduced at this point in the docs, so I'd either omit this or say something like "the last positional argument"

**Comment by ViralBShah** [comment]
@PaulSoderlind Can you update this PR as per the comments and rebase to master?

**Comment by PaulSoderlind** [comment]
@fredrikekre Any chance you can help out again? It would be much appreciated. (I am not really good at this.)

**Comment by PaulSoderlind** [comment]
rebased as #35725

**Pull request closed without merging**